receiver/jaegerreceiver/factory.go

```diff
@@ -82,7 +82,7 @@ func (f *Factory) CustomUnmarshaler() receiver.CustomUnmarshaler {
 		// these protocols were excluded during normal loading and we need to add defaults for them
 		vSub := v.Sub(viperKey)
 		if vSub == nil {
-			return fmt.Errorf("Jaeger receiver config is empty")
+			return fmt.Errorf("empty config for Jaeger receiver")
```

Comment:
Had to change this because strict-check does not like error messages that start with upper-case.